brew-script/action.yml

```diff
@@ -13,5 +13,19 @@ inputs:
     description: Ruby script to execute
     required: true
 runs:
-  using: node12
-  main: main.js
+  using: composite
+  steps:
+    - run: |
+        cat << 'EOF' > /tmp/brew-script-$GITHUB_SHA
+        ${{inputs.script}}
+        EOF
+      shell: bash
+    - run: brew ruby /tmp/brew-script-$GITHUB_SHA
+      shell: bash
+      env:
+        HOMEBREW_GITHUB_API_TOKEN: ${{inputs.token}}
+        HOMEBREW_NO_ENV_FILTERING: 1
+        HOMEBREW_NO_AUTO_UPDATE: 1
+        HOMEBREW_NO_ANALYTICS: 1
+        HOMEBREW_COLOR: 1
+        RUBYOPT: -W0
```

Comment:
Why set this?

Reply:
So one does not see messages like this:
```
/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/language/python.rb:8: warning: Insecure world writable dir /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/shims/scm in PATH, mode 040777

```

Reply:
We should fix that message either in Homebrew or, ideally, in the environment we execute this in.

Reply:
https://github.com/Homebrew/brew/pull/8407

Deleted in: https://github.com/Homebrew/actions/commit/f4b98344cb593dc08583251f91d6701719a0c873